README.md

```diff
@@ -6,24 +6,11 @@
 Install the module with: `npm install to-rgba-array`
```

Comment:
I'd like us to invest more time in READMEs. Our readme's are very poor (having reviewed all of the one's created in the past financial year).

I'd like to see at a minimum:

* Why
  * what is the purpose of this module? What problem does it solve? At least a few sentences. Some modules which are used to support bigger systems of other packages are very difficult to understand why they exist, and how they plug in to a bigger system.
  * If it's specific to a product, then discuss where it's used, and why it's used.

* Usage
  * several examples of usage. Usually copying and pasting code from the tests and making the code standalone suffices.

* API
  * spend a few moments documenting the full public API, the options, etc. Copying the code from the JSDoc comments from the module and reformatting them helps. For a good example of API documentation see [https://github.com/Level/levelup#api](node-levelup) also a module that I wrote: [https://github.com/eugeneware/pathdb](pathdb). The more you do it the easier it becomes. Docs are some of the highest value things we can create.
* Removal of all the standard boilerplate (eg. all the `.awesome` stuff). Ideally our tests should check the readme and fail if all the boilerplate is there and sections haven't been written.